1.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>breadcrumbs</title>
+    <link rel="stylesheet" href="1.css">
+</head>
+<body>
+    <nav class="nav">
+        <ul class="nav__inner breadcrumbs breadcrumbs--horizontal">
+            <li class="breadcrumbs__path-elem path-elem"><a href="#">Home\</a></li>
+            <li class="breadcrumbs__path-elem path-elem"><a href="#">Projects\</a></li>
+            <li class="breadcrumbs__path-elem path-elem"><a href="#">My Project\</a></li>
+        </ul>
+    </nav>   

```

Comment:
сделать через один блок, т.е один компонент, это значит описать один класс и его элементы.
также описать и у лабы 2

```suggestion
    <nav class="breadcrumb">
        <ol class="breadcrumb__inner">
            <li class="breadcrumb__item"><a class="breadcrumb__link" href="#">Home\</a></li>
            <li class="breadcrumb__item"><a class="breadcrumb__link" href="#">Projects\</a></li>
            <li class="breadcrumb__item"><a class="breadcrumb__link" href="#">My Project\</a></li>
        </ol>
    </nav>   
```